src/github.rs

```diff
@@ -1046,42 +1045,94 @@ pub struct Query<'a> {
 }
 
+fn quote_reply(markdown: &str) -> String {
+    if markdown.is_empty() {
+        String::from("*No content*")
+    } else {
+        format!("\n\t> {}", markdown.replace("\n", "\n\t> "))
+    }
+}
+
 #[async_trait]
 impl<'q> IssuesQuery for Query<'q> {
     async fn query<'a>(
         &'a self,
         repo: &'a Repository,
+        include_fcp_details: bool,
         client: &'a GithubClient,
     ) -> anyhow::Result<Vec<crate::actions::IssueDecorator>> {
         let issues = repo
             .get_issues(&client, self)
             .await
             .with_context(|| "Unable to get issues.")?;
-
         let issues_decorator: Vec<_> = issues
-            .iter()
-            .map(|issue| crate::actions::IssueDecorator {
-                title: issue.title.clone(),
-                number: issue.number,
-                html_url: issue.html_url.clone(),
-                repo_name: repo.name().to_owned(),
-                labels: issue
-                    .labels
-                    .iter()
-                    .map(|l| l.name.as_ref())
-                    .collect::<Vec<_>>()
-                    .join(", "),
-                assignees: issue
-                    .assignees
-                    .iter()
-                    .map(|u| u.login.as_ref())
-                    .collect::<Vec<_>>()
-                    .join(", "),
-                updated_at: crate::actions::to_human(issue.updated_at),
+            .into_iter()
+            .map(|issue| async move {
+                // TODO: this seems like a *really* bad approach, but I'm not sure how to make it better
+                let fcp_map = crate::rfcbot::get_all_fcps().await?;
+                let fcp_details = if include_fcp_details {
+                    let repository_name = if let Some(repo) = issue.repository.get() {
+                        repo.repository.clone()
+                    } else {
+                        let re = regex::Regex::new("https://github.com/rust-lang/|/").unwrap();
+                        let split = re.split(&issue.html_url).collect::<Vec<&str>>();
+                        split[1].to_string()
+                    };
+                    let key = format!(
+                        "rust-lang/{}:{}:{}",
+                        repository_name, issue.number, issue.title,
+                    );
+
+                    if let Some(fcp) = fcp_map.get(&key) {
+                        let bot_tracking_comment_html_url = format!(
+                            "{}#issuecomment-{}",
+                            issue.html_url, fcp.fcp.fk_bot_tracking_comment
+                        );
+                        let bot_tracking_comment_content = quote_reply(&fcp.status_comment.body);
+                        let fk_initiating_comment = fcp.fcp.fk_initiating_comment;
+                        let initiating_comment_html_url =
+                            format!("{}#issuecomment-{}", issue.html_url, fk_initiating_comment,);
```

Comment:
could just use the `comment.html_url` since we have to call `event.get_issue` anyway 

(however, this call will likely not be necessary if we make upstream changes to rfcbot)